mvn-extension/src/main/java/org/arquillian/smart/testing/mvn/ext/ModifiedPomExporter.java

```diff
@@ -0,0 +1,40 @@
+package org.arquillian.smart.testing.mvn.ext;
+
+import java.io.File;
+import java.io.IOException;
+import java.io.StringWriter;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import org.apache.maven.model.Model;
+import org.apache.maven.model.io.xpp3.MavenXpp3Writer;
+import org.arquillian.smart.testing.Logger;
+
+class ModifiedPomExporter {
+
+    static Logger logger = Logger.getLogger(ModifiedPomExporter.class);
+
+    private static String SMART_TESTING_POM_FILE = "smart-testing-pom.xml";
+
+    static void showPom(Model model) {
```

Comment:
Why is it still called `showPom` - it was called this way in the first version because it was printing it out to the console among other logs, but this name doesn't really reflect what is happening under the hood now.